dockstore-client/src/main/java/io/dockstore/client/cli/nested/NotificationsClients/NotificationsClient.java

```diff
@@ -30,16 +31,23 @@
 
 
     public NotificationsClient(String hookURL, String uuid) {
-        this.hookURL = hookURL;
-        this.uuid = uuid;
         boolean invalidHookURL = (hookURL == null || hookURL.isEmpty());
         boolean invalidUUID = (uuid == null || uuid.isEmpty());
-        if (invalidHookURL || invalidUUID) {
-            disabled = true;
+        if (invalidHookURL) {
             if (!invalidUUID) {
                 System.err.println("Notifications UUID is specified but no notifications webhook URL found in config file");
             }
+            disabled = true;
```

Comment:
Get rid of disabled. As discussed, just fail fast and early as possible if a UUID is specified without a webhook URL

Reply:
That disabled variable is used to determine whether or not notification messages are supposed to be sent (when there is a webhook URL regardless of UUID).

Reply:
Ah ok, then I may be looking at the wrong line.
Can you point me toward where the fast fail is being done?

Reply:
When the UUID is defined but the webhook url is undefined, the only extra thing that happens is the above System.err.println which is simply a warning.  Whether it's supposed to disabled or not was determined beforehand.

Reply:
Changing to abort launch ASAP...